k2/python/k2/fsa_algo.py

```diff
@@ -63,17 +63,17 @@ def top_sort(fsa: Fsa) -> Fsa:
 
 
 def intersect(a_fsa: Fsa, b_fsa: Fsa) -> Fsa:
-    '''Compute the intersection of two FSAs.
+    '''Compute the intersection of two FSAs on CPU.
 
     Note:
       We will arc_sort the input FSAs internally if they are not
```

Comment:
This would be quite slow if done repeatedly for the same inputs.  I think it might be better to just die if one of the inputs is not arc-sorted(?), and make arc-sorting a separate algorithm?

Reply:
Fixed.